### ISSUE TITLE
consider fixing invalid escape sequences

### ISSUE THREAD

**Issue by RonnyPfannschmidt** [asottile/pyupgrade#47]
```
$ python2 -W once -c 'print("\d test")'
\d test
$ python3 -W once -c 'print("\d test")'
<string>:1: DeprecationWarning: invalid escape sequence \d
\d test
$ python3 -W once -c 'print(r"\d test")'
\d test
```

**Comment by asottile** [comment]
Coincidentally I'm already working on this! There's a lot of edge cases but I think it's doable

**Comment by asottile** [comment]
This has been released as part of v1.10.0
